previews-cy-103

steiningermona opened the pull request.

steiningermona committed:
finish text and image preview, WIP icon preview

steiningermona committed:
finish text and image preview, WIP icon preview

steiningermona committed:
finish previews

steiningermona committed:
finish previews

diegomart2000 commented:
@steiningermona I don't understand what is introduced in this PR. There is code already reviewed coming from other PRs.

diegomart2000 reviewed (changes requested):
@steiningermona please rebase from dev

steiningermona committed:
Merge branch 'previews-cy-103' of https://github.com/caisy-io/league into previews-cy-103

Commit:
chore: clean up flag duplication

chore: clean up flag duplication

Commit:
fix: hm turns out preview-flag-round does equal

fix: hm turns out preview-flag-round does equal flag and that can also be used for flag-rounded 🐛

Commit:
Merge remote-tracking branch 'origin/dev' into previews-cy-103

diegomart2000 commented:
CLOSING this as it is a duplicate
